### ISSUE TITLE
Can't enter dates

### ISSUE THREAD

**Issue by fschon** [fooloomanzoo/datetime-input#3]
I really can't understand how to enter dates with this component. I've tried all the demos on webcomponents.org and even integrated it into my Vaadin 10 app, but entering dates just creates nonsense results. Firstly the tab sequence is strange, and then when entering a day, the year gets filled with rubbish numbers. When entering a year and then the month, the year will suddenly change. Its unpredictable and perhaps I'm really daft, but I just can't understand how to enter proper data into this component.

If it would work, then it would be a super solution.

**Comment by fooloomanzoo** [comment]
Could you please give an example of what you did try to enter to which field or property and what the outcome was? What excactly did not work? The demo on webcomponents.org, the github demo or in vaadin?

**Comment by fschon** [comment]
I tried the demos on webcomponents and on my own vaadin app - couldn't fidn
the demos on github. Any input creates non-sensical output. Just try type
theyear 1998 and then a month and things go wrong. Heres a link to one of
the demos: https://www.webcomponents.org/element/fooloomanzoo/datetime-input

In this case I just clicked in the year field, typed '1998' and got
06/29/1998. Then in the month I entered '04' and the year chnages to 2003!
If I try backspacing all sorts of things change.
